project/Build.scala

```diff
@@ -370,9 +370,12 @@ object Build {
             Seq(s"-Xplugin:$jar")
           }
         }
-      )
+      ).withScalaJUnitMixinPlugin
     }
 
+    def withScalaJUnitMixinPlugin: Project =
+      project.settings(ScalaJUnitMixinPlugin.projectSettings)
```

Comment:
Ah it's also used in `testSuiteJVM`. Then maybe just make it a `scalaJUnitMixinPluginSetting` of type `Setting[_]`, which can be included in `.settings()`.